execution-core/Cargo.toml

```
@@ -57,3 +62,5 @@ groth16 = [
 
 # Enables std feature for dusk-plonk
 std = ["dusk-plonk/std"]
+
+serde_support = ["serde", "bs58", "serde_json"]
```

```suggestion
serde = ["serde", "bs58", "serde_json"]
```
Many crates expose serde support on their own crates through a feature flag named the same as the crate, so I would prefer to rename this to just "serde" as a flag